init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/loader/Loader.java

```diff
@@ -13,6 +15,7 @@
 
 @Component
 public class Loader implements CommandLineRunner {
+    private final Logger logger = LoggerFactory.getLogger(CommandLineRunner.class);
```

Comment:
use the more standard approach - private static final Logger LOGGER ?? Theres a little bit of a mix in the codebase tbf..

Reply:
I moved everything to non  static, all except places that must have static loggers (i.e. static methods)